core/libraries/rest_api/CalculatedModelFields.php

```diff
@@ -55,6 +55,7 @@ protected function generateNewMapping()
         $event_calculations_class = $rest_api_calculations_namespace . 'Event';
         $datetime_calculations_class = $rest_api_calculations_namespace . 'Datetime';
         $registration_class = $rest_api_calculations_namespace . 'Registration';
+        $attendee_class = $rest_api_calculations_namespace . 'Attendee';
```

Comment:
Also, I wonder if this change should be made in a branch off of `master` and then it can be reviewed by Mike and merged upstream into GB/master and then this branch?  Although its a change being implemented in this branch, it does impact the usual rest response so probably should be isolated in its own pull.

Reply:
can do

Reply:
sure... I'll extract that today

Reply:
plz see #602